src/files.ts

```diff
@@ -134,11 +134,31 @@ export function parseFileIgnores(path: string): string[] {
 
 export function getGlobPatterns(supportedFiles: SupportedFiles): string[] {
   return [
-    ...supportedFiles.extensions.map(e => `*${e}`),
+    ...supportedFiles.extensions.map(e => `*${generateAllCaseGlobPattern(e)}`),
     ...supportedFiles.configFiles.filter(e => !EXCLUDED_NAMES.includes(e)),
```

Comment:
q: Should these be case sensitive or case insensitive?

Reply:
This is a good question. It's case sensitive currently. Making these case insensitive would be a different problem to solve and out of this PR scope, since the change would be not only in `code-client`.